cli/flox/doc/flox-push.md

```diff
@@ -50,10 +50,10 @@ FloxHub with local changes to the environment.
 :   Directory to push the environment from (default: current directory).
 
 `-o`, `--owner`
-:   Owner to push push environment to (default: current user).
+:   FloxHub owner to push environment to (default: current FloxHub user).
 
 `-f`, `--force`
-:   forceably overwrite the remote copy of the environment.
+:   Forceably overwrite the remote copy of the environment.
```

Comment:
**suggestion:** "Forceable" is British English. We use American English everywhere; thus, for consistency, this should be "Forcibly."

Reply:
🇺🇸